rapidash.go

```diff
@@ -122,10 +122,12 @@ type LastLevelCacheOption struct {
 }
 
 type TagOption struct {
-	server         string
-	expiration     time.Duration
-	lockExpiration time.Duration
-	ignoreStash    bool
+	server          string
+	expiration      time.Duration
+	lockExpiration  time.Duration
+	ignoreStash     bool
+	optimisticLock  bool
```

Comment:
The problem I mentioned;

> after this PR, it has falsy options for cache control by default.

can be solved with pointer reference, it enables nil evaluation.
thanks golang.